src/uct/ib/base/ib_md.c

```diff
@@ -1185,6 +1185,27 @@ static uct_md_ops_t UCS_V_UNUSED uct_ib_md_global_odp_ops = {
     .detect_memory_type = ucs_empty_function_return_unsupported,
 };
 
+int uct_ib_check_device_exists(struct ibv_device *device)
+{
+    /* Enough place to hold the full path */
+    char device_path[IBV_SYSFS_NAME_MAX + 16];
```

Comment:
can we use `device->dev_path`?

Reply:
I'm not sure about that.

`device->dev_path` is a symlink to the infiniband class device, which is a dir (e.g `/sys/class/infiniband_verbs/uverbs0 -> ../../devices/pci0000:00/0000:00:02.0/0000:04:00.0/infiniband_verbs/uverbs0`). It exists even in docker, and doesn't allow to determine, whether a device is accessible (unlike using `/dev/infiniband/uverbs0`).

Reply:
i see
maybe reuse IBV_SYSFS_PATH_MAX ?
or use dynamic string_buffer?